code/modules/jobs/job_types/roguetown/adventurer/types/combat/warrior.dm

```diff
@@ -310,3 +311,91 @@
+
+			var/legs = list(
+				"Chain Chausses"	= /obj/item/clothing/under/roguetown/chainlegs/iron,
+				"Chain Kilt"		= /obj/item/clothing/under/roguetown/chainlegs/iron/kilt
+				)
+			var/legschoice = input("Choose your Pants.", "TAKE UP PANTS") as anything in legs
+			pants = legs[legschoice]
+
+			gloves = /obj/item/clothing/gloves/roguetown/chain/iron
+			neck = /obj/item/clothing/neck/roguetown/bevor/iron
+			wrists = /obj/item/clothing/wrists/roguetown/bracers/iron
+			shoes = /obj/item/clothing/shoes/roguetown/boots/armor/iron
+			belt = /obj/item/storage/belt/rogue/leather/battleskirt/black
+			backl = /obj/item/storage/backpack/rogue/satchel
+			beltl = /obj/item/flashlight/flare/torch/lantern
+			backpack_contents = list(
+				/obj/item/storage/belt/rogue/pouch/coins/poor = 1,
+				/obj/item/recipe_book/survival = 1,
+				)
+			H.adjust_skillrank(/datum/skill/combat/polearms, 2, TRUE)
+			H.adjust_skillrank(/datum/skill/combat/maces, 2, TRUE)
+			H.adjust_skillrank(/datum/skill/combat/axes, 2, TRUE)
+			H.adjust_skillrank(/datum/skill/combat/swords, 2, TRUE)
+			H.adjust_skillrank(/datum/skill/combat/shields, 2, TRUE)
+			H.adjust_skillrank(/datum/skill/combat/whipsflails, 2, TRUE)
+			H.adjust_skillrank(/datum/skill/combat/wrestling, 3, TRUE)
+			H.adjust_skillrank(/datum/skill/misc/swimming, 1, TRUE)
+			H.adjust_skillrank(/datum/skill/combat/unarmed, 3, TRUE)
+			H.adjust_skillrank(/datum/skill/misc/athletics, 3, TRUE)
+			H.adjust_skillrank(/datum/skill/misc/climbing, 2, TRUE)
+			H.adjust_skillrank(/datum/skill/misc/reading, 1, TRUE)
+			H.dna.species.soundpack_m = new /datum/voicepack/male/warrior()
+			H.set_blindness(0)
+			ADD_TRAIT(H, TRAIT_HEAVYARMOR, TRAIT_GENERIC)
+			var/weapons = list("Executioner's Sword","Warhammer + Shield","Flail + Shield","Lucerne","Greataxe")
+			var/weapon_choice = input("Choose your weapon.", "TAKE UP ARMS") as anything in weapons
+			switch(weapon_choice)
+				if("Executioner's Sword")
+					H.adjust_skillrank(/datum/skill/combat/swords, 1, TRUE)
+					r_hand = /obj/item/rogueweapon/sword/long/exe
+				if("Warhammer + Shield")
+					H.adjust_skillrank(/datum/skill/combat/maces, 1, TRUE)
+					H.adjust_skillrank(/datum/skill/combat/shields, 1, TRUE)
+					beltr = /obj/item/rogueweapon/mace/warhammer
+					backr = /obj/item/rogueweapon/shield/iron
+				if("Flail + Shield")
+					H.adjust_skillrank(/datum/skill/combat/whipsflails, 1, TRUE)
+					H.adjust_skillrank(/datum/skill/combat/shields, 1, TRUE)
+					beltr = /obj/item/rogueweapon/flail
+					backr = /obj/item/rogueweapon/shield/iron
+				if("Lucerne")
+					H.adjust_skillrank(/datum/skill/combat/polearms, 1, TRUE)
+					r_hand = /obj/item/rogueweapon/eaglebeak/lucerne
+					backr = /obj/item/rogueweapon/scabbard/gwstrap
+				if("Greataxe")
+					H.adjust_skillrank(/datum/skill/combat/axes, 1, TRUE)
+					r_hand = /obj/item/rogueweapon/greataxe
+					backr = /obj/item/rogueweapon/scabbard/gwstrap
```

Comment:
adjust_skillrank_up_to(<skill>, 3, TRUE)

unless you want these guys to pick a weapon virtue and potentially get Expert. (They shouldn't in this case, but I'd rather keep the standard going)

Reply:
I am 99% sure virtues are applied after the loadouts. The other warrior subclasses also use adjust_skillrank and we do not have any master-skilled battlemaster incidents afaik.